modules/@ergonode/product-templates/src/components/Template/ProductDesigner/AttributeElementContent.vue

```diff
@@ -8,9 +8,12 @@
         @mouseover.native="onMouseOver"
         @mouseout.native="onMouseOut">
         <div class="element-content__icon">
-            <Component
-                :is="attributeIconComponent.component"
-                v-bind="attributeIconComponent.props" />
+            <template v-for="(formComponent, i) in typeIconComponent">
+                <Component
+                    :is="formComponent.component"
+                    :key="i"
```

Comment:
to clarify we are using index in every template, let's keep one convention

Reply:
the `index` was already used in the higher scope